project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -24,7 +28,7 @@ It is strongly recommended that you submit your request far enough in advance to
 approval before making relevant expenditures. Review of requests will take at least 72
 hours and may take considerably longer. Note, with limited funds available, approval
 is not guaranteed. If a request is rejected, reimbursement will not be issued. The
-Foundation cannot pre-pay for travel expenditures.
+Foundation cannot pre*pay for travel expenditures.
```

Comment:
```suggestion
Foundation cannot pre-pay for travel expenditures.
```

Reply:
I have already fixed those in the last commit

Reply:
This one still isn’t fixed. Perhaps you need to push?

Reply:
i just pushed it again as it showed okay on my screen